src/Tizen.NUI/src/public/WebView/WebViewPageLoadErrorEventArgs.cs

```diff
@@ -95,10 +94,11 @@ public enum LoadErrorCode
         }
 
         private WebView _webView;
+
         /// <summary>
         /// The view for displaying webpages.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 8 </since_tizen>
         public WebView WebView
```

Comment:
I think this will not be necessary : WebViewPageLoadErrorEventArgs.WebView

In C#, all event callbacks have a first argument sender which is WebView in WebView event

Reply:
removed.